src/HomePage.js

```diff
@@ -13,22 +14,73 @@ HomePage.propTypes = {
   onSelectGenre: PropTypes.func
 }
 
+
 export default function HomePage({concerts, onHeartClick, genres, onSelectGenre, onDeleteClick, selectedGenre}) {
+  
+  const [activeSort, setActiveSort] = useState('Latest added')
+
+  console.log(activeSort)
+  let sortedConcerts = concerts.slice()
+
+  function compare(a, b) {
```

Comment:
Ich würde diese Hilfsfunktionen eher ans Ende der Komponenten setzen. Wenn es generische Hilfsfunktionen wie die compare sind, könnten die sogar in eine utils-datei ausgelagert werden.